main.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2016 Table Top Inventing
+# All rights reserved.
+
+import cgi
+import json
+import urllib
+import webapp2
+from google.appengine.ext import ndb
+
+
+class measurePair(ndb.Model):
+    name = ndb.StringProperty()
+    value = ndb.FloatProperty()
+
+class userMorphology(ndb.Model):
+    emailID = ndb.StringProperty()
+    mPairs = ndb.StructuredProperty(measurePair, repeated=True)
+    date = ndb.DateTimeProperty(auto_now_add=True)
+    @classmethod
+    def query_morph(cls, user_key):
+        return cls.query(ancestor=user_key)
+
+class Profile(ndb.Model):
+    emailID = ndb.StringProperty()
+    firstName = ndb.StringProperty()
+    lastName = ndb.StringProperty()
+    userMeasurements = ndb.JsonProperty()
+    date = ndb.DateTimeProperty(auto_now_add=True)
+    @classmethod
+    def query_profile(cls, user_key):
+        return cls.query(ancestor=user_key)
+
+# This class handles both the post AND get methods for Profiles
+class manageProfile(webapp2.RequestHandler):
+    def post(self):
+        # This section needed to parse the userMeasurements section of the post body
+        j = json.loads(self.request.body)
+        uM = j['userMeasurements']  # this is a list of dict objectts
+        uPair = [measurePair(**item) for item in uM]  # this steps through the dictionary items from the dictionary list (uM), then it unpacks the dictionary item into measured pair, but it works because the input is assumed to have the form [{"name": "waistCircumference", "value": 32},{"name": "neckCircumference", "value": 16}]
+        userMorph = userMorphology(parent = ndb.Key("Profile", j['emailID'] or "*notitle*"),
+            emailID = j['emailID'], mPairs = uPair)
+
+#        userMorph.mPairs = uPair    # hand uPair list of objects to the mPairs (repeated = True) structure container
+        print "Morpho:  ", userMorph.mPairs
+        userMorph.put()
+
+        thisProfile = Profile(parent=ndb.Key("Profile",
+                                           j['emailID'] or "*notitle*"),
```

Comment:
Add a TODO that 'emailID' is a required field. Then, in a later change, we can check and enforce it, sending back a 400 Bad Request code https://en.wikipedia.org/wiki/List_of_HTTP_status_codes